src/popup/router/pages/Index.vue

```diff
@@ -139,40 +180,45 @@ export default {
       margin-bottom: 8px;
     }
 
-    span {
-      display: block;
+    .heading {
+      @extend %face-sans-18-medium;
 
-      &.heading {
-        @extend %face-sans-16-medium;
+      @include mixins.flex(center);
 
-        color: variables.$color-blue;
-        padding: 5.5px 0;
-        margin: 0 auto;
-      }
+      line-height: 22.5px;
```

Comment:
We should not use values that are smaller than a pixel. Also for line heights its better to use em.

Reply:
Kept px for now. I borrowed it from the Figma design.